### PR TITLE
add italian tokenization

### PR DESCRIPTION
**Pull request by GiacomoLeoneMaria**
# What does this PR do ?

Add italian tokenization

### CODE REVIEW

File: nemo/collections/common/tokenizers/text_to_speech/tts_tokenizers.py [hunk 1]
```diff
@@ -694,10 +723,7 @@ def __init__(
         pad_with_space=False,
         text_preprocessing_func=chinese_text_preprocessing,
     ):
-        """
-        Chinese phoneme-based tokenizer.
-        Note: This tokenizer for now covers Chinese phonemes/tones and English letters because our dataset contains
-              both Chinese and English graphemes.
+        """Chinese phoneme-based tokenizer.
```
Comment (XuesongYang) [inline nemo/collections/common/tokenizers/text_to_speech/tts_tokenizers.py:726]: just to confirm: is this removal intentional?

**Review by XuesongYang — COMMENTED** [review]
LGTM. Could you please also add some unit tests for italian tokenizer?

* https://github.com/NVIDIA/NeMo/blob/main/tests/collections/common/tokenizers/text_to_speech/test_tokenizer_utils.py
* https://github.com/NVIDIA/NeMo/blob/main/tests/collections/common/tokenizers/text_to_speech/test_tts_tokenizers.py

File: nemo/collections/common/tokenizers/text_to_speech/tts_tokenizers.py (hunk 1 above)
Comment (GiacomoLeoneMaria) [reply nemo/collections/common/tokenizers/text_to_speech/tts_tokenizers.py:726]: it was my mistake, thank you for noticing it!

**Review by XuesongYang — APPROVED** [review]
